src/blocks/text.js

```diff
@@ -32,6 +32,48 @@ const extractColorClasses = (html) => {
     };
 };
 
+const convertDynamicImages = (html) => {
+    const frag = JSDOM.fragment(html);
+    let placeholders = frag.querySelectorAll('span[type="dynamicImage"]');
+    let content = html;
+    if (placeholders.length > 0 ) {
+        placeholders.forEach((ph) => {
+            let image = {
+                alt: ph.getAttribute('alt'),
+                src: ph.getAttribute('src'),
+                condition: ph.getAttribute('condition'),
+                width: ph.getAttribute('width'),
+                height: ph.getAttribute('height'),
+            };
+
+            let imgDom = dom.window.document.createElement('img');
+            imgDom.setAttribute('src', image.src || '');
+            imgDom.setAttribute('alt', image.alt || '');
+            if (image.width && image.width !== '') {
+                imgDom.setAttribute('width', image.width || '');
+            }
+            if (image.height && image.height !== '') {
+                imgDom.setAttribute('height', image.height || '');
+            }
+
+            if (image.condition) {
+                ph.parentNode.insertBefore(dom.window.document.createTextNode(`{{#if ${image.condition}}}`), ph);
+                ph.parentNode.insertBefore(imgDom, ph);
+                ph.parentNode.insertBefore(dom.window.document.createTextNode('{{/if}}'), ph);
```

Comment:
Would there be an "else" support. One of the use cases that we had was "show image of the business but if there is no image show its name". Would that be supported in the dynamicImage, or would it be achievable in some other fashion ?

Reply:
I'd rather not add this at this point. Let's see how this would look look. The else statement can be easily provided with inverted if.